changelog/unreleased/issue-122

```diff
@@ -0,0 +1,8 @@
+Enhancement: Verify uploaded files
+
+rest-server now verifies that the hash of content of uploaded files matches
```

Comment:
Perhaps mention 'by default' and 'opt-out with --no-verify-upload`?

Reply:
Good idea!